CADETProcess/optimization/pymooAdapter.py

```diff
@@ -272,7 +272,11 @@ def _do(self, problem, X, **kwargs):
         X_new = None
         for i, ind in enumerate(X):
             if not self.optimization_problem.check_individual(
-                    ind, untransform=True, get_dependent_values=True):
+                    ind,
+                    untransform=True,
+                    get_dependent_values=True,
+                    check_nonlinear_constraints=False,
+                    ):
                 if X_new is None:
```

Comment:
Good question, I'm not sure about this; I believe it was not silent before. But maybe it would make sense to silence it. Right now, if too many new individuals need to be created, it can give me some insight as to the well-posedness of my problem but it's not something that happens too often. And maybe I should check this somewhere else?

But I think for now, I will just leave it "as is".